scenarios/post_action_shut_down.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+import subprocess
+import logging
+import time
+
+
+def run(cmd):
+    out = ""
+    try:
+        output = subprocess.Popen(
+            cmd, shell=True, universal_newlines=True, stdout=subprocess.PIPE, stderr=subprocess.STDOUT
+        )
+        (out, err) = output.communicate()
+    except Exception as e:
+        logging.error("Failed to run %s, error: %s" % (cmd, e))
+    return out
+
+
+wait_duration = 10
+timeout = 900
+counter = 0
+not_ready = run("oc get nodes --no-headers | grep 'NotReady' | wc -l").rstrip()
```

Comment:
Maybe we should add checks for API reachability and cluster operators status to cover the case where cerberus is not deployed as part of the run? Thoughts?

Reply:
Should be added now with doing `oc get co`